docs/core/linux-prerequisites.md

```diff
@@ -114,7 +136,19 @@ To install .NET Core on supported RHEL versions:
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-To ensure you have the latest installation information, follow the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-current) for supported RHEL versions.
+**.NET Core 2.0**
+
+ Install .NET Core 2.x on supported RHEL versions:
```

Comment:
2.x or 2.0? I'm not seeing the 2.1 links here

Reply:
2.0